src/json_printer.sql

```diff
@@ -55,40 +70,65 @@ package body "JSON_PRINTER" as
       return str;
     end if;
   end llcheck;  
-
-  function escapeString(str varchar2) return varchar2 as
-    sb varchar2(32767) := '';
-    buf varchar2(40);
-    num number;
+  
+  -- escapes a single character. 
+  function escapeChar(ch char) return varchar2 deterministic is
```

Comment:
Sure. As I said, it is just aestetics. that function,being backed up by a cache, gets called just once for each distinct character met in the whole lifetime of the program. its performance is irrelevant.

 just give me a sec.. Just have to figure out how to do it... As I said I am new to github...